### ISSUE TITLE
Terraform apply for openid-connect plugin always shows changes

### ISSUE THREAD

**Issue by kylerivakong** [Kong/terraform-provider-konnect#87]
Terraform provider keeps updating openid-connect plugin even when no change is made to the plugin template. 

This makes it very difficult to track the actual changes in the terraform template deployment.

Steps to reproduce:

Attached main.tf file with oidc plugin config.

Added a valid issuer URL for line 116 of the main.tf file.

terraform init
terraform apply: to complete the initial creation of the plugin
terraform apply: subsequent applies with the same .tf file with no changes to it has terraform reporting that the plugin was changed.

Will receive the follow output for the subsequent applies:

```
konnect_gateway_plugin_openid_connect.openid_connect: Modifying... [id=dea80add-3333-4cdc-b21b-e28d9c280449]
konnect_gateway_plugin_openid_connect.openid_connect: Modifications complete after 0s [id=dea80add-3333-4cdc-b21b-e28d9c280449]

Apply complete! Resources: 0 added, 1 changed, 0 destroyed.

```

[oidc-terraform.zip](https://github.com/user-attachments/files/17367238/oidc-terraform.zip)

**Comment by mheap** [comment]
This is resolved by #99, which will go out today or tomorrow

**Comment by mheap** [comment]
This is released as 2.0.0 as it's a breaking change
